.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push]
```

Comment:
The azure pipelines trigger was set for commits/PRs on master only. Did you want to change this behavior?
Otherwise might look like? (push will trigger after merge, but if you want it before merge during PR but not on the commit of the other branch)
```
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

Reply:
Ah yeah that's more like what I want